src/operations/find_and_modify.ts

```diff
@@ -29,7 +29,7 @@ export interface FindOneAndDeleteOptions extends CommandOperationOptions {
   sort?: Sort;
   /** Map of parameter names and values that can be accessed using $$var (requires MongoDB 5.0). */
   let?: Document;
-  /** Return the raw result document instead of the ModifyResult */
+  /** Return the ModifyResult instead of the modified document. Defaults to true. */
```

Comment:
Can we add a note that this will default to false in the next major version?

Reply:
Updated.